logbook-spring-boot-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookAutoConfiguration.java

```diff
@@ -149,18 +149,55 @@ private Collection<BodyFilter> mergeWithTruncation(List<BodyFilter> bodyFilters)
     }
 
     private Predicate<HttpRequest> mergeWithExcludes(final Predicate<HttpRequest> predicate) {
-        return properties.getExclude().stream()
+        final Predicate<HttpRequest> deprecatedPredicate = properties.getExclude() // backward compatibility for old config
+                .stream()
                 .map(Conditions::requestTo)
                 .map(Predicate::negate)
                 .reduce(predicate, Predicate::and);
+
+        return properties.getPredicate().getExclude()
+                .stream()
+                .map(this::convertToPredicate)
+                .map(Predicate::negate)
+                .reduce(predicate, Predicate::and)
+                .and(deprecatedPredicate);
```

Comment:
I suggest simplifying this function as follows:

```suggestion
        return Stream.concat(
                        properties.getExclude() // backwards compatibility for deprecated config
                                .stream()
                                .map(Conditions::requestTo),
                        properties.getPredicate().getExclude()
                                .stream()
                                .map(this::convertToPredicate)
                )
                .map(Predicate::negate)
                .reduce(predicate, Predicate::and);
```

Reply:
This looks much better! Applied in https://github.com/zalando/logbook/pull/1606/commits/7f73fce9ab7031c67f676b2b7b809302d1859a35